### DISCUSSION TITLE
Indoor Coverage Map

### DISCUSSION THREAD

**Discussion by nourhan0106** [NVlabs/sionna#511] [Q&A]
I am using Sionna to simulate a factory setup with multiple APs. I have the APs placed in some rooms but not all. I am not sure why when I am plotting the coverage map, there is no coverage in the other rooms. Seems like the wave penetration is not included. I am not sure what I am doing wrong.

**Comment by SebastianCa** [comment]
Hi @nourhan0106,
There is currently no support for refraction in Sionna (see #414) which explains why your simulations show no coverage in the other rooms.
